examples/example-all-fail/expected_results.json

```
@@ -0,0 +1,28 @@
+{
+  "version" : "v2",
```

```suggestion
  "version" : "2",
```

This comment also applies to the other files.